task_spec/spec/spec_helper_acceptance.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
-require 'puppet'
```

Comment:
We'll be able to go back to this when we unvendor puppet, right?

Reply:
yes

Reply:
I think a comment reminding us of that would be nice.